src/app.js

```diff
@@ -1,14 +1,18 @@
 const express = require('express');
 const session = require('express-session');
 const passport = require('passport');
-require('dotenv').config();
 const swaggerUi = require('swagger-ui-express');
+require('dotenv').config();
 
 const Database = require('./db');
 const authRouter = require('./routes/auth');
 const errorHandler = require('./middlewares/errorHandler');
 const logger = require('./utils/logger');
 const swaggerDocument = require('./api-documentation.json');
+const projectRoutes = require('./routes/project');
+const postRoutes = require('./routes/post');
+const commentRoutes = require('./routes/comment');
+const profileRoutes = require('./routes/profile');
```

Comment:
What about routes for My Profiles?
`const myProfileRoutes = app.use('/api/myprofile', myProfileRoutes);`